### PR TITLE
bsc#1099749-adds & changes per bug attachment

### PR DESCRIPTION
**Pull request by csymons-suse**
changes and additions per attachment to bug
decision forthcoming 2018-07-10

### CODE REVIEW

**Comment by sknorr** [comment]
The changes in this PR are dependent on code changes that might happen around July 12, whether they will go in will be determined on July 10.

File: xml/art_migration.xml [hunk 1]
```diff
@@ -1065,6 +1065,32 @@ sudo zypper dup --from Cloud</screen>
...
+     add</command> command, for example:
+    </para>
+    <screen>&prompt.ardana;openstack role add -/-user <replaceable>USER
+    NAME</replaceable> -/-project <replaceable>PROJECT NAME</replaceable>
+    cinder_admin</screen>
+    <para>
+     Role assignments should be carefully considered. In an environment which
+     requires separation of duties, each operator should be assigned only the
+     roles they require to perform their duties and should use their own user
+     to perform their administrative tasks. for more information see the <xref
```
Comment (bryanstephenson) [inline xml/art_migration.xml]: "for" -> "For"

**Comment by sknorr** [comment]
@csymons-suse I assume @bryanstephenson's message means that the code went in (??). If so, I guess you could remove the comments from you commit. (Of course, it's hackweek, i.e. feel free not to do this today/tomorrow.)

**Comment by csymons-suse** [comment]
Content has been updated according to bsc#1099749 notes. Should be reviewed for accuracy.

**Pull request merged**